### PR TITLE
enables formatting check

### PR DESCRIPTION
**Pull request by kjezek**
this PR re-enables formatting check that was temporarily suspended. 
Plus formatting is fixed

### CODE REVIEW

**Comment by Mike-CZ** [comment]
The `license_header.txt` file should be also modified.

**Comment by kjezek** [comment]
yes, the format checker was failing because of this subtle change. 

since the initial headers were not added manually, not by using the script, I would not put relation between this PR and the `license_header.txt` file

**Comment by Mike-CZ** [comment]
> yes, the format checker was failing because of this subtle change.
> 
> since the initial headers were not added manually, not by using the script, I would not put relation between this PR and the `license_header.txt` file

Intereseting! I do partially agree on this. The license header will need to be changed because of this PR anyway.

**Review by HerbertJordan — APPROVED** [review]

File: cpp/archive/archive.h [hunk 1]
```diff
@@ -6,8 +6,8 @@
  *
  * Change Date: 2028-4-16
  *
- * On the date above, in accordance with the Business Source License, use
- * of this software will be governed by the GNU Lesser General Public Licence v3.
+ * On the date above, in accordance with the Business Source License, use of
+ * this software will be governed by the GNU Lesser General Public Licence v3.
```
[the remark this answers is not in the document]
Comment (kjezek) [reply cpp/archive/archive.h]: done

**Review by HerbertJordan — APPROVED** [review]
Thanks!

**Comment by HerbertJordan** [comment]
> The `license_header.txt` file should be also modified. What is the cause of the error? It seems a bit off that it was caused just by a comment starting by letter "of".

I think the problem was simply that the last line of the license text was too long.
In the C++ format style there is a maximum line length.

**Pull request merged**